common/help-on-spot-models/src/utils/Database.ts

```diff
@@ -1,41 +1,57 @@
-import { Connection, createConnection } from 'typeorm';
-import {PostgresConnectionOptions} from "typeorm/driver/postgres/PostgresConnectionOptions";
+import { Connection, ConnectionManager, ConnectionOptions, createConnection, getConnectionManager } from 'typeorm'
+import {ormConfig} from "../ormConfig";
 import User from "../entity/User";
-import Address from "../entity/Address";
 import Qualification from "../entity/Qualification";
 import Organisation from "../entity/Organisation";
+import Address from "../entity/Address";
 import Request from "../entity/Request";
 import RequestResponse from "../entity/RequestResponse";
-import {ormConfig} from "../ormConfig";
 import {QualificationsMigration1586981133000} from "../migration/qualificationsMigration";
 
 export class Database {
-  private connection?: Connection;
+  private connectionManager: ConnectionManager
+
+  constructor() {
+    this.connectionManager = getConnectionManager()
+  }
+
+  public async getConnection(): Promise<Connection> {
+    const CONNECTION_NAME = `default`
```

Comment:
Maybe add This as optional param to the function?
lambdas would be able to create multiple connections and name their own connection to prevent further conflicts?

Reply:
I don't know if it is wanted to let lambdas create their own connection names, but we can do it.